### ISSUE TITLE
 转ONNX提示少OP  including einsum,tril_triu,

### ISSUE THREAD

**Issue by sue2xlh** [PaddlePaddle/Paddle2ONNX#919]
您好，我在将模型转为ONNX格式的时候，报了以下问题：
![image](https://user-images.githubusercontent.com/7824188/192965862-0b5d368a-d84f-4c40-ac57-08ab12082b30.png)

我看了一下代码，可能是由于使用了paddle.einsum 和 paddle.tril的缘故。
在想有没有什么办法解决上面的报错？

**Comment by yeliang2258** [comment]
paddle2onnx应该是没有支持这两个OP，您看看有没有办法用其他更常用的OP来组合实现这两个OP的功能。
这个后面我们会排期支持，但是这个周期要长一些。

**Comment by sue2xlh** [comment]
嗯嗯，好滴，谢谢答复。

**Comment by sue2xlh** [comment]
不好意思，又打扰了，tril 这个怎么能用其他常用OP组合替代呢？比如这个  paddle.tensor.tril(paddle.ones_like(logits), diagonal=-1)  。想了很久，没有想到合适的组合。想请教一下您，有什么合适的组合推荐吗

**Comment by yeliang2258** [comment]
你看看可以先用numpy生成这个矩阵，然后使用paddle.to_tensor直接转为paddle的tensor吗？

**Comment by sue2xlh** [comment]
paddle2onnx的过程中，  logits的类型是 <class 'paddle.fluid.framework.Variable'>， paddle.to_tensor(np.tril(paddle.ones_like(logits).numpy(), k=-1))     我这样写 会报错

**Comment by GUSHUMING** [comment]
> paddle2onnx的过程中， logits的类型是 <class 'paddle.fluid.framework.Variable'>， paddle.to_tensor(np.tril(paddle.ones_like(logits).numpy(), k=-1)) 我这样写 会报错

请问解决了吗

**Comment by sue2xlh** [comment]
> > paddle2onnx的过程中， logits的类型是 <class 'paddle.fluid.framework.Variable'>， paddle.to_tensor(np.tril(paddle.ones_like(logits).numpy(), k=-1)) 我这样写 会报错
> 
> 请问解决了吗

没解决，换成用pytorch了

**Comment by Vivecccccc** [comment]
是uie 吗？我也遇到了完全相同的算子不支持

**Comment by Zheng-Bicheng** [comment]
* https://github.com/PaddlePaddle/Paddle2ONNX/pull/1314

**Comment by Zheng-Bicheng** [comment]
* https://github.com/PaddlePaddle/Paddle2ONNX/pull/1269
